modules/identity/pages/ldap-synchronizer.adoc

```diff
@@ -711,19 +711,21 @@ Caused by: org.apache.http.client.HttpResponseException: status code: 404
 ----
 
 *Root cause:*
-The LDAP Synchronizer is using the Engine HTTP API of the product but it's not active in your configuration.
+The LDAP Synchronizer is using the Engine HTTP API of the product, but it's not active in your configuration.
 
 *Solutions:*
 
-*On-premise solution*: Configure the ZIP bundle you've downloaded from the bonitasoft Customer Service Center
-* Edit the `/server/webapps/bonita/WEB-INF/web.xml` file and make sure these lines are not commented:
-[source,xml]
-----
- <servlet>
-        <servlet-name>HttpAPIServlet</servlet-name>
-        <servlet-class>org.bonitasoft.engine.api.internal.servlet.HttpAPIServlet</servlet-class>
- </servlet>
-----
-*Docker image solution:* Run the docker image with *-e HTTP_API=true*
-* For example: `docker run -e HTTP_API=true -e REST_API_DYN_AUTH_CHECKS=false --name bonita -v ./bonita-lic/:/opt/bonita_lic/ -h localhost -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:7.9.2`
-* See the page xref:ROOT:bonita-docker-installation.adoc#_http_api[Deploy Bonita Runtime with Docker] for more information.
+*On-premise solution*:
+
+Configure the Bonita Tomcat bundle you have downloaded from the Bonitasoft Customer Service Center to activate
+the Engine HTTP API by: 
+
+* either add a Java system property `-Dhttp.api=true` to file `setEnv.[sh|bat]` inside tomcat bundle (in folder `setup/tomcat_templates/`)
+* or set an environment variable `HTTP_API=true` before launching Bonita Tomcat bundle
+
+*Docker image solution:*
+
+Run the docker image with *-e HTTP_API=true*
+
+* For example: `docker run -e HTTP_API=true -e REST_API_DYN_AUTH_CHECKS=false --name bonita -v ./bonita-lic/:/opt/bonita_lic/ -h localhost -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:pass:a[{bonitaVersion}]`
```

Comment:
That will fail because of the missing `HTTP_PASSWORD` (I'm not sure about the variable name)